code/modules/atmospherics/machinery/portable/portable_atmospherics.dm

```diff
@@ -62,6 +62,11 @@
 	anchored = 1 //Prevent movement
 	return 1
 
+/obj/machinery/portable_atmospherics/Move()

```

Comment:
we all know how atmos is super finicky, please make sure this meme doesn't result in some stupid things happening